users/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from django.contrib.auth.models import User
+
+
+class Profile(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    bio = models.TextField(max_length=500, blank=True)
+    date_of_birth = models.DateField(null=True, blank=True)
```

Comment:
@ahmadx16 For models which are meant to be updated by user or user's action should have created and modified date for reporting purposes.